File: Drinks.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    
    int sum = 0;
    for (int i = 0; i < n; i++) {
        int x;
        cin >> x;
        sum += x;
    }
    
    double result = (double) sum / n;
    cout << result << endl;
    
    return 0;
}

File: Drinks.java
import java.util.Scanner;

public class Drinks {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int n = scanner.nextInt();
        int sum = 0;
        
        for (int i = 0; i < n; i++) {
            sum += scanner.nextInt();
        }
        
        double result = (double) sum / n;
        System.out.println(result);
        
        scanner.close();
    }
}